src/Components/Home/ProductRemovedAlert.js

```diff
@@ -0,0 +1,23 @@
+import { Fragment } from "react";
+import { Alert } from "react-bootstrap";
+
+const ProductRemovedAlert = (props) => {
+  return (
+    <Fragment>
+      <Alert show={props.showForgetAlert} variant="success" className="my-1">
+        <p>
+          <i className="far fa-smile"></i> The product has been removed from the
+          list. Good job!
+        </p>
+      </Alert>
+      <Alert show={props.showBoughtAlert} variant="danger" className="my-1">
```

Comment:
Could we change the alert when a product is bought after the waiting period has elapsed?

I think the current Alert of red colour and frown makes sense for if the User skipped the waiting period. 

However I think waiting 30 days, answering the consideration questions and then deciding to buy the product should be a success event in the app with a more encouraging message. Perhaps a message along the lines of "Congratulations on considering this purchase carefully. Go ahead without regrets."

Reply:
we don't track if a user skipped a waiting period ... 😕

Reply:
This comment can be a case for further research with users.

Reply:
> we don't track if a user skipped a waiting period ... 😕

not yet, but I feel there should be an option for a user to do that in a later iteration

Reply:
Such functionality is called "auditing" and it's hell of a work. I'm not convinced. We may have include this in user testing after Milestone 3, or even later.